INSTALL.md

```diff
@@ -252,6 +265,23 @@ It might be necessary to restart postgresql: `sudo service postgresql restart`.
 
 Furthermore, the `pg_hba.conf` file can be modified to change authentication type for local users to trust (rather than, e.g., md5) but we haven't tested this solution.
 
+#### `FATAL: password authentification failed for user "postgres"`
+If you get a traceback similar to this one when making a test environment:
```

Comment:
@aliu104 I wouldn't include the traceback. It's specific to Jorge and if we started including traceback examples for every scenario the documentation would become unreasonably long. I would include just the error message encountered, and your solution. Note that your proposed solution lowers security, and that's something other users should be made aware of. I would include a reference to the PostgreSQL documentation on how to modify the pg_hba.conf file and I would change, "modifying the `pg_hba.conf`" to include the full path to the file on Ubuntu. I would also note that the path may be different for different distributions.